OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/PushBroadcastReceiver.java

```diff
@@ -114,6 +115,14 @@ protected void onPushReceived(Context context, PushMessage pushMessage) {
         this.showNotification(context, pushMessage, builder.build());
     }
 
+    private boolean notificationsEnabled(Context context){
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
```

Comment:
Is this correct / a user may still disable notifications for an app on lower API levels